### PR TITLE
BZ1600199:Corrected role clusteradmin to cluster-admin

### PR DESCRIPTION
**Pull request by prashant-redhat**
@openshift/team-documentation: Peer review needed.

BZ1600199: Corrected the role clusteradmin to cluster-admin and made changes to according to the style guide.

### CODE REVIEW

File: admin_guide/manage_rbac.adoc [hunk 1]
```diff
@@ -758,23 +758,23 @@ $ oc create clusterrole podviewonly --verb=get --resource=pod
...
+to a user, using a local role binding, making it appear that this user has the
+privileges of a cluster administrator. This is not the case. Binding the `cluster-admin`
+to a certain project is more like a super administrator for that
+project grants the permissions of the cluster role
 `admin`, plus a few additional permissions like the ability to edit rate limits.
-This can appear especially confusing via the web console UI, which does not list
-cluster role bindings (which are bound to true cluster administrators). However, it
-does list local role bindings (which could be used to locally bind `clusteradmin`).
+This can appear confusing especially via the web console UI, that does not list
+cluster role bindings, that are bound to true cluster administrators. However, it
```
Comment (kalexand-rh) [inline admin_guide/manage_rbac.adoc]: s/bindings, that/bindings that

**Review by kalexand-rh — COMMENTED** [review]
A few more tweaks and this is good. :)

Which branches does this apply to?

**Comment by vikram-redhat** [comment]
@kalexand-rh I think we should restrict the changes from this to our default N-2 model. That is, enterprise-3.10, 3.9, and 3.7 (and of course, 3.11). I have added those flags.

**Comment by vikram-redhat** [comment]
@prashant-redhat - the next step would be to take the feedback provided by @kalexand-rh and update your PR so her changes are incorporated.

**Comment by prashant-redhat** [comment]
@kalexand-rh - updated the PR

**Comment by kalexand-rh** [comment]
@prashant-redhat, after you rebase, you can merge.

**Comment by prashant-redhat** [comment]
@kalexand-rh: Please have another look. It's been rebased and I have updated it as well. Needs to be merged.

**Comment by kalexand-rh** [comment]
@prashant-redhat, if you have permission to merge, you should do it yourself. IDK if you do, though, so I'll merge it in for you. :)

**Pull request merged**